plugin/src/App/StatusPages/Connected.lua

```diff
@@ -367,13 +378,59 @@ function ConnectedPage:render()
 				height = self.changeDrawerHeight,
 				layoutOrder = 5,
 
+				showSourceDiff = function(oldSource: string, newSource: string)
+					self:setState({
+						showingSourceDiff = true,
+						oldSource = oldSource,
+						newSource = newSource,
+					})
+				end,
+
 				onClose = function()
 					self.changeDrawerMotor:setGoal(Flipper.Spring.new(0, {
 						frequency = 4,
 						dampingRatio = 1,
 					}))
 				end,
 			}),
+
+			SourceDiff = e(StudioPluginGui, {
+				id = "Rojo_ConnectedSourceDiff",
+				title = "Source diff",
+				active = self.state.showingSourceDiff,
+
+				initDockState = Enum.InitialDockState.Float,
+				initEnabled = true,
```

Comment:
What does this prop do? This looks unused.

Reply:
Huh. You're right. I copied it from App, I guess it did something once upon a time and was never cleaned up. Good eye!